app/main.py

```diff
@@ -16,4 +16,18 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {"mutable": [], "immutable": []}
+
+for i in [lucky_number,
+          pi,
+          one_is_a_prime_number,
+          name,
+          my_favourite_films,
+          profile_info,
+          marks,
+          collection_of_coins]:
+    if isinstance(i, (list, dict, set)):
+        sorted_variables["mutable"].append(i)
+    else:
+        sorted_variables["immutable"].append(i)
```

Comment:
u just need to place variables into lists by right keys
don't complicate script